numpy/random/mtrand.pyi

```diff
@@ -0,0 +1,132 @@
+import builtins
+from typing import overload, Optional, Sequence, Tuple, TypeVar, Union
+
+from numpy import ndarray
+
+_Size = Union[int, Tuple[int, ...]]
+_T = TypeVar('_T')
+
+class RandomState:
+    def __init__(self, state: Optional[Union[int, ndarray]] = ...) -> None: ...
+    def bytes(self, length: int) -> builtins.bytes: ...
+    def dirichlet(self, alpha: Sequence[float], size: Optional[_Size] = ...
+                  ) -> ndarray: ...
+
+    @overload
+    def choice(self, a: int, *,
+               replace: bool = ..., p: Optional[Sequence[float]] = ...
+               ) -> int: ...
+    @overload
+    def choice(self, a: Sequence[_T], *,
+               replace: bool = ..., p: Optional[Sequence[float]] = ...
+               ) -> _T: ...
```

Comment:
`a` gets cast to an ndarray, so the return value is the corresponding NumPy scalar type.